experts/MAP.md

```diff
@@ -0,0 +1,6 @@
+# Expert Map
+
+This document contains the different areas of the compiler and the people that
+are experts in each area. While the expert map is being fleshed out, [you can find it on HackMD][map].
```

Comment:
I would add:

---

This map is intended to help you figure out who you should ask for help if you have questions about how some area works.